### PR TITLE
Completed Homework-01

### PR DESCRIPTION
**Pull request by SayleeMJ**

### CODE REVIEW

**Comment by dstad418** [comment]
Great use of equality operators and arrow functions! They met both the Clean Code & AirB&B style guides. 

The only recommendation I can make is putting strings in single-quotations instead.

**Comment by SayleeMJ** [comment]
> Great use of equality operators and arrow functions! They met both the Clean Code & AirB&B style guides.
> 
> The only recommendation I can make is putting strings in single-quotations instead.

I made the changes. I hope this is correct. Let me know if I need to make more changes

**Comment by dstad418** [comment]
Looks great to me now!

**Review by dstad418 — APPROVED** [review]
Comments left 10/5

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+  const displayResult = document.getElementById("result");
+  displayResult.className = resultColorClass;
+  displayResult.textContent = resultTextString;
+};
+const elem = document.querySelector("input");
+
+const handleInput = () => {
+  const input = parseInt(elem.value);
+
+  if (!Number.isNaN(input)) {
```
Comment (lebenpdx) [inline hw1/02-palindrome.js]: Rule [19.9](https://github.com/airbnb/javascript#whitespace--padded-blocks) states that blocks should not be padded with whitespace

File: hw1/02-palindrome.js [hunk 2]
```diff
@@ -1,3 +1,33 @@
-const elem = document.querySelector('input');
+function isPalindrome(value) {
+  const stringInput = value.toString();
+  const reversedInput = stringInput.split("").reverse().join("");
```
Comment (lebenpdx) [inline hw1/02-palindrome.js]: Rule [19.6](https://github.com/airbnb/javascript#whitespace--chains) states that long method chains(2 or more) should be broken up with indentation, with a leading dot.

**Review by lebenpdx — COMMENTED** [review]
Methodology looks fantastic. Just some very minor style issues.

**Comment by mattsloe** [comment]
Great work! Same thing with the single quotes for strings and also the style guide recommends that parseInt() be used with a value for the radix.

**Pull request merged**